README.rst

```diff
@@ -20,6 +20,7 @@
 Addons-Server
 =============
 
+foo
```

Comment:
Like Kung Fu?

Reply:
Doh, I was test-generating git diffs and failed to not commit that change :rofl: